packages/search/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
 ## 1.0.0
 
 - Add Search component based on `calypso/components/search`
+
+## 1.0.1
```

Comment:
New changelog entries should be at the beginning, so that the latest version is first.